gamestonk_terminal/stocks/discovery/yahoofinance_model.py

```diff
@@ -44,3 +44,32 @@ def get_undervalued() -> pd.DataFrame:
     data = pd.read_html(requests.get(url).text)[0]
     data = data.iloc[:, :-1]
     return data
+
+
+def get_gtech() -> pd.DataFrame:
+    """Get Yahoo Finance growth technology stocks
+
+    Returns
+    -------
+    pd.DataFrame
+        Growth technology stocks
+    """
+    url = "https://finance.yahoo.com/screener/predefined/growth_technology_stocks"
+
+    data = pd.read_html(requests.get(url).text)[0]
+    data = data.iloc[:, :-1]
+    return data
+
+
+def get_active() -> pd.DataFrame:
+    """Get Yahoo Finance most active stocks
```

Comment:
Similar as above. I.e. description from here https://finance.yahoo.com/screener with the [Source: Yahoo Finance] at the end